src/ontology/uberon-edit.obo

```diff
@@ -10379,7 +10379,7 @@ intersection_of: composed_primarily_of CHEBI:17029 ! chitin
 relationship: develops_from UBERON:0007376 ! outer epithelium
 relationship: only_in_taxon NCBITaxon:6656 ! Arthropoda
 relationship: part_of UBERON:0002416 ! integumental system
-property_value: editor_note "TODO - split from exoskeleton? Also: See above sf item to see if this belongs in GO" xsd:string
```

Comment:
The QC is failing because of this unexpected change. I don't know what could cause this. When I open Uberon in Protégé, I see this "new" annotation.

There are only a few cases:
<img width="1188" alt="Screenshot 2023-10-13 at 10 01 55" src="https://github.com/obophenotype/uberon/assets/2208124/e5c649b0-7cb0-4cdd-9d86-3abfa9df46e9">

<img width="647" alt="Screenshot 2023-10-13 at 10 01 42" src="https://github.com/obophenotype/uberon/assets/2208124/d63107e2-d697-4432-88dd-b056434b39df">

Reply:
@matentzn @gouttegd @balhoff

Reply:
Maybe: editors note were previously not included when converting an obo file to owl format. Solution:

- add "editors note" as permitted property
- remove editors note with robot remove during preprocessing

Reply:
I’d like to be sure we understand what’s happening before hiding the problem under the rug…

We have 910 `editor_note` annotations on classes in Uberon (not counting the annotations on axioms).

They are correctly preserved in the -edit file when switching the serialisation format to the one used in OWL API 4.5.26 (whether the switch is done with Protégé or with `robot convert`). That is, all instances of `property_note: editor_note "..."` in `uberon-edit.obo` are preserved.

But:

When preprocessing the -edit file to produce `tmp/uberon-preprocessed.owl` (in functional syntax), only 890 of those annotations are correctly rendered as `http://purl.obolibrary.org/obo/IAO_0000116`, as in:

```
AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000116> <http://purl.obolibrary.org/obo/UBERON_0000021> "Mammary glands […] here.")
```

but 25 are rendered as `http://www.geneontology.org/formats/oboInOwl#0000116`, as in:

```
AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#0000116> <http://purl.obolibrary.org/obo/UBERON_0001001> "TODO - split from exoskeleton? Also: See above sf item to see if this belongs in GO")
```

This does not cause issues to save the file in functional syntax, but does prevent saving the file in RDF/XML, because the XML serialiser tries to write that as `<oboInOwl:0000116>`, and IIRC XML names cannot start with a number.

Now what I’d like to know is _why_ those 25 annotations are rendered incorrectly???

Reply:
It becomes interesting… We also have one `IAO:0000116` annotation on an axiom:

```
[Term]
id: UBERON:0003674
[…]
relationship: has_part UBERON:0006844 {IAO:0000116="cardinality not allowed with transitivity in OWL", count="1"}
```

That annotation is _also_ rendered incorrectly as `http://www.geneontology.org/formats/oboInOwl#0000116`. And it seems to be the cause of the entire issue: removing that axiom annotation makes all the other `editor_note` annotations on classes to be correctly rendered as `http://purl.obolibrary.org/obo/IAO_0000116`.

Reply:
We should be able to use `IAO:0000116` in axiom annotations, as we actually have several other instances where this does not cause any problem, e.g.:

```
relationship: part_of UBERON:0001016 {IAO:0000116="we do not commit to CNS or PNS to allow for flexibility with this class"} ! nervous system
[…]
is_a: UBERON:0012112 {IAO:0000116="consider changing to derivation relationship", source="FMA"} ! ingested food
[…]
relationship: part_of UBERON:0000030 {IAO:0000116="check this"} ! lamina propria
```

All those annotations are correctly rendered as `http://purl.obolibrary.org/obo/IAO_0000116`. So I am unsure what the problem is with

```
relationship: has_part UBERON:0006844 {IAO:0000116="cardinality not allowed with transitivity in OWL", count="1"}
```

Reply:
I'm not sure what's going wrong. But that annotation that's causing the problem has two annotations:
- `IAO:0000116="cardinality not allowed with transitivity in OWL"`
- `count="1"`

`count` is not declared as an annotation property in the edit file. Could you give it a `Typeref` and see if that helps?

Reply:
No, it doesn’t change anything. And:
* the `IAO:0000116` annotation on UBERON:0003674 is incorrectly translated as `http://www.geneontology.org/formats/oboInOwl#0000116` even when the `count=1` annotation is absent;
* elsewhere in the ontology, we also have this:

```
relationship: has_part UBERON:0006844 {IAO:0000116="cardinality not allowed with transitivity in OWL", UBPROP:0000008="some premolars may have 3 cusps", min_count="2"} ! cusp of tooth
```

where the `IAO:0000116` annotation is correctly translated as `http://purl.obolibrary.org/obo/IAO_0000116` despite the fact that `min_count` is also not declared (and therefore translated under `oboInOwl`).